addons/ui/stringtable.xml

```diff
@@ -209,5 +209,17 @@
             <Chinese>聯絡伺服器的管理員</Chinese>
             <French>Contacter l'administrateur du serveur.</French>
         </Key>
+        <Key ID="STR_CBA_UI_LobbyManager">
+            <English>Lobby Manager</English>
+            <German>Lobby Manager</German>
+            <Polish>Menadżer Lobby</Polish>
+        </Key>
+        <Key ID="STR_CBA_UI_AdjustingGroupOrder">
+            <English>Adjusting group order...</English>
+            <German>Reihenfolge der Gruppen wird angepasst ...</German>
+        </Key>
+        <Key ID="STR_CBA_UI_LobbyManager_Warning">
+            <English>Back up mission before using this tool.</English>
```

Comment:
```suggestion
            <English>Back up mission before using this tool.</English>
            <German>Erstelle eine Sicherungskopie der Mission, bevor du den Lobby Manager verwendest.</German>
```

Reply:
Text is slightly too long for available space. Can you shorten it?

<details>
<summary>Image</summary>

![lobby_manager_2](https://user-images.githubusercontent.com/34453221/120103927-fc8a4880-c11f-11eb-9867-e5465d98cf10.png)


</details>

Reply:
`<German>Erstelle vor Gebrauch eine Sicherungskopie.</German>`

Short enough?

Reply:
Yes, that should work. Can you update the suggestion with that?

Reply:
Done ☺️